### ISSUE TITLE
USWDS date picker for public side (Design Audit)

### ISSUE THREAD

**Issue by cantsin** [18F/crt-portal#807]
### Date 
- [ ] Can we add the USWDS date picker? We use it on the CRT side

#### Currently
![Screen Shot 2020-11-06 at 2.25.36 PM.png](https://images.zenhubusercontent.com/5846f186b9dc3c6220230bf1/e03c8c0f-36c9-40a8-99e6-67ad2ef68528)

#### Could add (_used in table filters_)
![Screen Shot 2020-11-06 at 2.26.25 PM.png](https://images.zenhubusercontent.com/5846f186b9dc3c6220230bf1/3ccf744a-1e41-4eac-b54b-85a782950c34)

**Comment by cantsin** [comment]
Also may merit discussion with the larger team @blaughman @randyabramson

**Comment by randyabramson** [comment]
Yup...I support using https://designsystem.digital.gov/form-controls/05-date-picker/

cc: @cantsin
